konnectd/ui/src/app.css

```diff
@@ -0,0 +1,69 @@
+/* additional css on top of kpop */
+html {
+  font-family: 'Open Sans', sans-serif;
+}
+
+strong {
+  font-weight: 600;
+}
+
+.oc-logo {
+  position: absolute;
+  top: -140px;
+  left: 50%;
+  height: 120px;
+  transform: translateX(-50%);
+}
+
+.oc-input {
+  background-color: white;
+  border: 0;
+  border-radius: 6px;
+  width: 300px;
+  height: 40px;
+  padding: 12px 8px;
+  box-sizing: border-box;
+  font-size: 1.125rem;
+}
+
+.oc-input:focus {
+  outline: none;
+}
+
+.oc-input + .oc-input {
+  margin-top: 10px;
+}
+
+.oc-button-primary {
+  /* Needs to be important to overwrite material-ui */
+  background-color: #4e85c8 !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)